sample_bird_workflow.yml

```diff
@@ -1,13 +1,13 @@
-name_array:
+names:
   - Input1
   - Output1
-alignments_array:
+alignment_sets:
   - { class: File, path: "/data/reddylab/gjohnson/scz/ORI_library_files_and_data/alignment_and_processing/alignments/Input1/Input1_lengthFiltered.bam" }
   - { class: File, path: "/data/reddylab/gjohnson/scz/ORI_library_files_and_data/alignment_and_processing/alignments/Output1/Output1.bam" }
```

Comment:
I was hoping we could use the sampled files instead of these multi GB ones for the sample `yml` file (since otherwise it becomes a bit hard to test and debug this). I will open an issue about this and we can work on it later.